src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -145,16 +146,58 @@ public function lightListAction(ProductFilters $filters, Request $request): Resp
      * @AdminSecurity("is_granted('create', request.get('_legacy_controller'))", message="You do not have permission to create this.")
      *
      * @param Request $request
+     * @param int $productId
      *
      * @return Response
      */
-    public function createAction(Request $request): Response
+    public function productShopsAction(Request $request, int $productId): Response
```

Comment:
shouldn't the action start with some "action keyword" like for example "chooseShopsAction" ?